confignode/src/main/java/org/apache/iotdb/confignode/manager/ConfigManager.java

```diff
@@ -48,36 +52,34 @@
 public class ConfigManager {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(ConfigManager.class);
+  private static final ConfigNodeConf conf = ConfigNodeDescriptor.getInstance().getConf();
 
   private IConsensus consensusImpl;
   private ConsensusGroupId consensusGroupId;
 
   private DeviceGroupHashExecutor hashExecutor;
 
-  @TestOnly
-  public ConfigManager(String hashExecutorClass, int deviceGroupCount) {
-    setHashExecutor(hashExecutorClass, deviceGroupCount);
```

Comment:
why delete?  we need only  test configManager.

Reply:
Because this constructor is useless...